### PR TITLE
Actually load data

### PR DESCRIPTION
**Pull request by brinxmat**
- Creates a lambda that accepts events
- Example of expected data is about 1/3 of this PR.
- Depends on other PRs, so maybe review these first (i.e. doesn't build in CI)

### CODE REVIEW

File: data-loading/build.gradle [hunk 1]
```diff
@@ -3,7 +3,16 @@ plugins {
...
+    implementation libs.aws.sdk2.lambda.events
+    implementation libs.aws.lambda.java.core
+    implementation libs.bundles.logging
 
+    testImplementation libs.nva.s3
+    testImplementation libs.jackson.databind
+    testImplementation libs.aws.sdk2.s3
+    testImplementation libs.nva.json
+    testImplementation libs.nva.commons.core
+    testImplementation libs.nva.testutils
```
Comment (brinxmat) [inline data-loading/build.gradle:15]: Some of these may be moved to `implementation` later

File: data-loading/src/main/java/no/sikt/nva/data/report/api/etl/DataLoader.java [hunk 2]
```diff
@@ -1,5 +1,20 @@
 package no.sikt.nva.data.report.api.etl;
 
-public class DataLoader {
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+public class DataLoader implements RequestHandler<SQSEvent, Void> {
```
Comment (brinxmat) [inline data-loading/src/main/java/no/sikt/nva/data/report/api/etl/DataLoader.java:9]: Does not do anything yet

File: data-loading/src/main/java/no/sikt/nva/data/report/api/etl/PersistedResourceMessage.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package no.sikt.nva.data.report.api.etl;
+
+import java.net.URI;
+
+public record PersistedResourceMessage(URI resourceFileUri) {
```
Comment (brinxmat) [inline data-loading/src/main/java/no/sikt/nva/data/report/api/etl/PersistedResourceMessage.java:5]: The expected blob-event

File: data-loading/src/test/java/no/sikt/nva/data/report/api/etl/DataLoaderTest.java [hunk 4]
```diff
@@ -1,11 +1,63 @@
...
 class DataLoaderTest {
+    public static final ObjectMapper objectMapper = JsonUtils.dtoObjectMapper;
+
+    private static final String BUCKET_NAME = "irrelevant";
+
+    private S3Driver s3Driver;
+    private Context context;
+
+    @BeforeEach
+    void setup() {
```
Comment (brinxmat) [inline data-loading/src/test/java/no/sikt/nva/data/report/api/etl/DataLoaderTest.java:32]: We expect a blob event, the S3-reference needs to exist

File: data-loading/src/test/java/no/sikt/nva/data/report/api/etl/DataLoaderTest.java [hunk 5]
```diff
@@ -1,11 +1,63 @@
...
+        var s3Client = new FakeS3Client();
+        s3Driver = new S3Driver(s3Client, BUCKET_NAME);
+        context = new FakeContext();
+    }
 
     @Test
-    void dataLoadingExists() {
+    void shouldLogStuff() {
```
Comment (brinxmat) [inline data-loading/src/test/java/no/sikt/nva/data/report/api/etl/DataLoaderTest.java:39]: We want logging, right?

**Review by truhacevkir — APPROVED** [review]

**Pull request merged**